packages/polymath-issuer/src/pages/ticker/components/ReserveTickerForm.js

```diff
@@ -24,6 +24,12 @@ const formSchema = validator.object().shape({
       /^[a-z0-9./-]+$/i,
```

Comment:
```suggestion
      /^[a-z0-9.-]+$/i,
```

Leftovers from boris' reign of terror

Reply:
Thanks @monitz87 didn't catch those